### ISSUE TITLE
WebPack-Error while starting Kibana with the plugin installed

### ISSUE THREAD

**Issue by FabianHaasjes** [nreese/ez-query#5]
Hi, 
first of all, iam currently running Kibana 5.5.2 while trying to install your plugin.

When i try to access Kibana i always get the same Error:
TypeError: __webpack_require__(...).register is not a function

Is this a compatibility issue with newer versions of Kibana? If so, do you intend to update the plugin to work with current releases of Kibana?

**Comment by nreese** [comment]
Yes, its a compatibility issue. Kibana 5.5 changed how some of its dependencies are exported - going from exporting an unnamed default to named exports. I will not have time to update the plugin for a little while but PRs are welcome
